packages/stack/src/navigators/createStackNavigator.tsx

```diff
@@ -17,11 +19,11 @@ import {
   StackNavigationEventMap,
 } from '../types';
 
-type Props = DefaultNavigatorOptions<StackNavigationOptions> &
+export type Props = DefaultNavigatorOptions<StackNavigationOptions> &
   StackRouterOptions &
   StackNavigationConfig;
 
-function StackNavigator({
+export function StackNavigator({
```

Comment:
I think exposing this is confusing (there's no the difference between this and `Stack.Navigator` to the user, which one should they use). Why do you need it?